labelspark/__init__.py

```diff
@@ -15,10 +14,10 @@ def create_dataset(client, spark_dataframe, dataset_name="Default"):
     # expects spark dataframe to have two columns: external_id, row_data
     # external_id is the asset name ex: "photo.jpg"
     # row_data is the URL to the asset
-    spark_dataframe = spark_dataframe.to_koalas()
+    spark_dataframe = spark_dataframe.to_pandas_on_spark()
     dataSet_new = client.create_dataset(name=dataset_name)
 
-    # ported Pandas code to koalas
+    # ported Pandas code to Pandas-on-Spark
```

Comment:
#ported Pandas code to Pandas-on-Spark (will use Koalas for legacy Spark versions)